### ISSUE TITLE
AdGuard VPN leaking IP when using the WinTUN driver for torrenting. 

### ISSUE THREAD

**Issue by davele739** [AdguardTeam/AdGuardVPNForWindows#676]
### Please answer the following questions for yourself before submitting an issue.

- [X] I checked the [knowledge base](https://adguard-vpn.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

2.3.1

### Environment

```markdown
- OS: Windows 11, 23H2 (OS build 22631.3593)
- Other network-level software: AdGuard ad blocker
```

### Which DNS server do you use?

AdGuard DNS

### Custom DNS

quic://4d21975e.d.adguard-dns.com

### Use Wintun

- [x] yes, I do

### Issue Details

Steps to reproduce:
1. Check the WinTUN option on AdGuard VPN
2. Make sure you have a torrent client 
3. Go to ipleak.net and select the torrent address activation option. 
4. Open the magnet link with your torrent client
5. Check the IPs that are displayed on the website. 
6. You will see your actual IP along with AdGuard's torrenting IPs, meaning that there is a leak

### Expected Behavior

The purpose of torrenting anonymously is to hide your IP address. Unfortunately, AdGuard VPN appears to leak your actual IP address when you use the WinTUN driver for torrenting.

### Actual Behavior

Actual IP is leaking. 

### Screenshots

<details><summary>Screenshot 1</summary>

 

</details>

**Comment by AlexandrPkhm** [comment]
Hi @davele739,

Unfortunately, we were unable to reproduce the issue. The IP address detected in the Torrent Address detection is not the same as the selected VPN location. However, this is because torrent traffic is redirected to a different server, and the detected IP does not represent the real client IP address. 

We kindly ask you to try installing the [latest Nightly version of AdGuard VPN](https://agrd.io/windows_vpn_nightly) and check if the issue is still reproducible. If this is the case, please send us the app logs. Here's what we need you to do:

1. Go to Settings → App settings → Logging level → **Record everything**;
2. Reproduce the problem and remember the exact time it was reproduced at. We will
need it to find the corresponding entries in the log file;
3. Open Support tab → **Export logs and system info**;
4. Send the archive to `devteam@adguard.com` and mention this issue number in the subject. Don't forget to include the time of reproduction.
5. Set the logging level back to **Record by default**.

Please let us know of the results, regardless of the outcome.

**Comment by davele739** [comment]
I understand. I resolved the issue by binding my torrent client to AdGuard VPN and disabling WebRTC, preventing any IP leaks. To confirm, I also downloaded Transmission (another torrent client) and verified that my actual ISP-provided IP address was not revealed while torrenting. This confirmed that the issue I previously experienced with qBittorrent was indeed with the torrent client itself, not the VPN.Thank you for confirming that others are not experiencing this issue. It helped me rule out any problems with my device or setup.
